Dockerfile

```
@@ -10,6 +10,9 @@ RUN mkdir -p /npm_cache && \
 
 USER node-red
 
-RUN npm install --save @alvarobc2412/status@latest
+RUN npm install --save @statuscompliance/status@latest  @statuscompliance/control-flow@latest  @statuscompliance/extraction@latest  @statuscompliance/filtering@latest @statuscompliance/integration@latest  @statuscompliance/logic@latest  @statuscompliance/validation@latest
```

```suggestion
RUN npm install --save @statuscompliance/status @statuscompliance/control-flow @statuscompliance/extraction  @statuscompliance/filtering @statuscompliance/integration @statuscompliance/logic @statuscompliance/validation
```

latest is already the default tag used by npm, so why make this unnecessarily verbose?